donations/apps.py

```diff
@@ -1,9 +1,7 @@
+# -*- coding: utf-8
 from django.apps import AppConfig
 
-class DonationConfig(AppConfig):
+
+class DonationsConfig(AppConfig):
     name = 'donations'
     verbose_name = "Django Donations"
-    def ready(self):
-        from .models import load_providers, load_frequencies
-        load_providers()
-        load_frequencies()
```

Comment:
Well if we want to keep the functionality of autoloading models based on settings then yes, but change the import to be `from .config_loaders import ...`

However I am not completely happy with the design so if we can think of/find a better way to do it, I am all ears :).
Eg We could just have migrations that create data required to make the library usable or just leave it blank.

Looking at the code again, providers seem more important as the class name is being held in the model, frequencies was just a helper to get started with minimum effort so could be removed.